### PR TITLE
feat: golden-response-rules

### PR DESCRIPTION
**Pull request by loekadam**
To try and make the output align more with the '7 gouden regels'.
Tried several levels of describing the rules, and several methods of implementation.
Current method results in output that is (based on survey results) more aligned with the rules.

### Types of descriptions gouden regels:

- Only the *rules*
- Rules and their description/components
- Only a description of the most important aspects

### Location of implementation

- `MELDING_HANDLING_GUIDELINES`
- `AGENTIC_AI_AGENT_PROMPT_PREFIX`
- `AGENTIC_AI_AGENT_PROMPT_FORMAT_INSTRUCTIONS`
    - As an additional action to rewrite

### Result

- Incorporation in the format instructions was most influential, but also added it to several other components.

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -152,7 +152,9 @@
 If you find useful information using the tools, provide that information in your Final Answer (in the language of the melding) while adhering to the above policies. 
 Start with "Ik heb je opmerking verzonden. Dit is wat ik je nu al kan vertellen." (in the language of the melding).
 If you cannot find any useful information, respond in your Final Answer with: "Ik heb je opmerking verzonden. Helaas heb ik geen relevante informatie gevonden om alvast met je te delen" (in the language of the melding).
-Don't thank the citizen for making a melding. Do thank them for being involved.
+Do not thank the citizen for making a melding. Do thank them for being involved.
```
Comment (ShaylaJan) [inline src/config.py:155]: Might not want to thank them at all (see golden rules). But for this PR it is ok.

File: src/central_agentic_agent.py
Comment (ShaylaJan) [inline src/central_agentic_agent.py:1]: Why did you add a new version of this file as well? The changes do not seems to relate to the goal of the PR.

Comment (loekadam) [reply src/central_agentic_agent.py:1]: Reason was because it now included the inputs for the survey, but maybe not necessary to include indeed.
Removed it
